source/frontend/src/features/mapSideBar/disposition/tabs/offersAndSale/dispositionSale/form/DispositionSaleFormYupSchema.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-template-curly-in-string */
+import * as yup from 'yup';
+
+export const DispositionSaleFormYupSchema = yup.object().shape({
```

Comment:
these are all strings, but in actuality a lot of these values are numbers or dates, please fix.
Make sure to enforce number size limits.
Also, it probably makes sense to limit the fiscal year to 4 digits.

Reply:
updated